### PR TITLE
Chitter PR

### PR DESCRIPTION
**Pull request by lacurnow**
# Your name
Lucinda Curnow
Please write your full name here to make it easier to find your pull request.

# User stories 

Please list which user stories you've implemented (delete the ones that don't apply).

- [X] User story 1: "I want to see all the messages (peeps) in a browser"
- [X] User story 2: "I want to post a message (peep) to chitter"
- [X] User story 3: "I want to see the date the message was posted"
- [ ] User story 4: "I want to see a list of peeps in reverse chronological order"
- [ ] User story 5: "I want to filter on a specific keyword"

# README checklist

Does your README contains instructions for

- [ ] how to install,
- [ ] how to run,
- [ ] and how to test your code?

[Here is a pill](https://github.com/makersacademy/course/blob/main/pills/readmes.md) that can help you write a great README!

### CODE REVIEW

**Comment by khushnoodasif** [comment]
From what I have observed this code accomplishes what is supposed to do.

**Comment by khushnoodasif** [comment]
- [x]  Was the code easy to understand? Yes
- [x]  Can the readability of the code be improved by smaller methods? Methods used were simplified to a great extent.
- [x]  Can the readability of the code be improved by different function/method or variable names? Readability is great but I would suggest using rubocop to see if there is anything with code that could be improved such as adding a newline after the end of code for better readability.
- [x]  Is the code located in the right file/folder/package? Yes

File: app.rb [hunk 1]
```diff
@@ -1,9 +1,22 @@
-require 'sinatra/base'
+require 'sinatra'
+require 'sinatra/reloader' if development?
+require './lib/peepviewer'
+require './lib/peep'
 
 class Chitter < Sinatra::Base
```
Comment (EdwardAndress) [inline app.rb:6]: You could probably remove the route for the test page

File: app.rb [hunk 2]
```diff
@@ -1,9 +1,22 @@
...
   get '/test' do
     'Test page'
   end
 
+  get '/view_peeps' do
```
Comment (EdwardAndress) [inline app.rb:11]: This is great but the naming is a little unconventional.

I'd expect the routes to be

`GET '/peeps'` for viewing all peeps
`POST '/peeps'` for creating a new peep

And I would expect the model to be called `Peep` so in the controller you'd do things like `Peep.all` and `Peep.create`.

There are also conventions for template names :)

`index` for the page that shows all the peeps

Comment (EdwardAndress) [reply app.rb:11]: Ah! I see that there _is_ a model called `Peep` and that you've implemented another class which takes care of the database interaction. This is a nice idea! I'd probably call that class something like `PeepRepository`

**Comment by EdwardAndress** [comment]
This is a nice solution! I particularly appreciate the fact that you've thought about code quality – this is reflected in your decision to implement the `PeepViewer` class. You've clearly demonstrated the learning objectives for this module.

**Pull request closed without merging**